grammar.js

```diff
@@ -266,6 +272,7 @@ module.exports = grammar({
     // users and tooling which don't think about `try`s as having a "then". Thus,
     // `try`s are essentially treated the same as any other expression.
     _expression_seq: ($) => repeat1(choice($._expression, $.try)),
+    _statement_seq: ($) => repeat1(choice($._statement, $.try)),
```

Comment:
Corresponding to the Gleam parser, we should probably delete `_expression_seq` as it's no longer used.  Also my big comment about `try` is now irrelevant as `try` has been fully removed, but let's save that for a separate PR.